### ISSUE TITLE
Primitive nodes with attributes aren't put into the schematic with those attributes

### ISSUE THREAD

**Issue by mtrberzi** [manifold-lang/manifold-frontend#70]
For example, `register = primitive node (initialValue: Bool, resetActiveHigh: Bool, resetAsynchronous: Bool, clockActiveHigh: Bool, clock: digitalIn, reset: digitalIn, in: digitalIn) -> (out: digitalOut);` only gets the `digitalIn` and `digitalOut` ports in the schematic, not the `Bool` attributes.

**Comment by maxqchen** [comment]
I tried reproducing this straight from a schematic -> serialize path and couldn't. After adding a bool attribute to the node type and a bool value to the node value, I got this:

```
 "in_node_name": {
      "attributes": {
        "inNodeAttr": "Bool"
      },
      "ports": {
        "in_port_name": "digital_in"
      }
    }
...
"nIN": {
      "type": "in_node_name",
      "attributes": {
        "inNodeAttr": "true"
      },
      "portAttrs": {
        "in_port_name": {}
      }
    },
```

**Comment by mtrberzi** [comment]
But is that in the schematic, or are you invoking the frontend? I think
this is a frontend problem.

**Comment by maxqchen** [comment]
Yeah that was built from the schematic. Just confirming it wasn't a core issue.
